libraries/mpxe-store/src/store.c

```diff
@@ -14,7 +14,9 @@
 #include "log.h"
 #include "stores.pb-c.h"
 
-#define MAX_STORE_COUNT 64
+#define MAX_STORE_COUNT 1000
```

Comment:
I think my point was that we shouldn't need that many since we shouldn't be initializing them all for all the i2c addresses etc., sorry should have clarified

Reply:
Sounds good. I'll just change it back to 64